processing/src/main/java/io/druid/query/aggregation/Aggregator.java

```diff
@@ -51,6 +54,15 @@ default double getDouble()
     return (double) getFloat();
   }
 
+  /**
+   * returns true if the Aggregator supports returning null values and the aggregated value is Null.
```

Comment:
For backward compat, makes sense to specify that if this is object aggregator, it may return isNull() = false even if get() returns null. And account for that at use sites.

Reply:
added.

Reply:
Please rewrite this doc according to the new policy (always false for logical object aggregators)

Reply:
I think returning always false is an implementation detail, not sure if that belongs to the API javadoc. 
Can you elaborate more on exactly what you need me to add here ?